benchmark/networkx_benchmark.py

```diff
@@ -1,13 +1,34 @@
+import numpy.random
+
 from benchmark.base_benchmark import Benchmark
 from networkx.readwrite.edgelist import read_edgelist
 from networkx.algorithms import community
 from benchmark.base_benchmark import AlgorithmNotFound
 from evaluation.networkx_evaluator import NetworkxEvaluator
+from benchmark.serialization.serialization import write_com_to_file, get_com_path
+from benchmark.utils import count_lines
+
 
 def girvan_newman(graph):
```

Comment:
Dit könntest du in eine Funktion packen anstatt 5 Funktionen zu schreiben... also eine Funktion wo du den entsprechenden Library-Call übergibst

Reply:
Hab ich versucht, machts wirlich dolle viel komplizierter und so haben wir mehr kontrolle und vor allem viel mehr leserlichkeit..

Reply:
Ich denk n zweizeiler ist nicht so kompliziert, aber ich kann es dann auch gerne machen

Reply:
`def run_networkx_algo(algo, *args, **kwargs):
    communities_iter = algo(*args, **kwargs)
    return list(communities_iter)`